packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -132,6 +132,35 @@ export type AreParamEqual<A extends (props: any) => JSX.Element | undefined, B e
 // @public
 export type AreTypeEqual<A, B> = A extends B ? (B extends A ? true : false) : false;
 
+// @beta
+export interface AtMentionDisplayOptions {
+    atMentionSuggestionRenderer?: (suggestion: AtMentionSuggestion) => JSX.Element;
+}
+
+// @internal
```

Comment:
If that is exposed in communication-react.api.md, it shouldn't be internal, either make it beta or remove it from here

Reply:
@PorterNan These are props for an internal component _AtMentionFlyout so I guess it should be internal instead of beta? By "remove it from here", do you mean manually remove it?  These are auto-generated for me when I ran `rush build -t communication-react`. I saw that other internal props (eg: _DrawerMenuProps) are declared the same way but somehow didn't show up here.